Assignment2/eduardogeuens-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,43 @@
+{
+	"@context":{
+		"re": "http://assignrent2b.com#"
+		"fullName":"re:fullName",
+		"horetown":"re:horetown",
+		"age":"re:age",
+		"almaMater":"re:almaMater",
+		"parentsName":"re:parentsName",
+		"subjects":"re:subjects",
+		"name":"subjects:name",
+		"teachers":"subjects:teachers",
+		"description":"subjects:description"

```

Comment:
These definitions work, but semantically they aren't ok. The best way to define them might be defining a new prefix for the subject concept and then use it to define their attributes.